lib/features/outline/OutlineProvider.ts

```diff
@@ -1,12 +1,11 @@
-import { Element } from '../../model/Types';
+import type { Element } from "../../model/Types";
```

Comment:
Do you mind undoing the `'` changes. We use single ticks in our code base.

Reply:
Oh absolutely, I didn't realize vscode mist have done that automatically

Reply:
done ✅